project/Dependencies.scala

```diff
@@ -0,0 +1,30 @@
+import sbt._
+
+object Dependencies {
+  val akkaV = "2.5.1"
+  val akkaHttpV = "10.0.6"
+  val jacksonV = "2.8.7"
+
+  val rootDependencies = Seq(
```

Comment:
Should we adopt the Rawls idiom where we define dependencies up front in vals? I personally find that nice to read.

https://github.com/broadinstitute/rawls/blob/develop/project/Dependencies.scala